vnext/Microsoft.ReactNative/Views/ViewViewManager.cpp

```diff
@@ -385,19 +395,35 @@ bool ViewViewManager::UpdateProperty(
     }
   }
 
-  if (auto view = pViewShadowNode->GetView().try_as<xaml::UIElement>()) {
+  return ret;
+}
+
+void ViewViewManager::OnPropertiesUpdated(ShadowNodeBase *node) {
+  auto *viewShadowNode = static_cast<ViewShadowNode *>(node);
+  auto panel = viewShadowNode->GetViewPanel();
+
+  if (panel.Background() == nullptr) {
+    // In XAML, a null background means no hit-test will happen.
+    // We actually want hit-testing to happen if the app has registered
+    // for mouse events, so detect that case and add a transparent brush.
+    if (viewShadowNode->IsHitTestBrushRequired()) {
+      panel.Background(EnsureTransparentBrush());
+    }
+    // Note:  Technically we could detect when the transparent brush is
+    // no longer needed, but this adds complexity and it can't hurt to
+    // keep it around, so not adding that code (yet).
```

Comment:
should we file a tracking issue for this? this would happen when e.g. we remove borderRadius from a view that had it?

Reply:
It would happen if the app unregisters all their mouse handlers after having them registered.  This seems unlikely and not worth optimizing so I won't bother tracking.

---
In reply to: [466854910](https://github.com/microsoft/react-native-windows/pull/5659#discussion_r466854910) [](ancestors = 466854910)